mp/src/game/client/momentum/ui/HUD/hud_mapfinished.cpp

```diff
@@ -336,10 +372,21 @@ void CHudMapFinishedDialog::SetVisible(bool bVisible)
     if (bVisible)
     {
         SetCurrentPage(0);
+        SetAlpha(255.0f);
 
         const auto pSpecUI = gViewPortInterface->FindPanelByName(PANEL_SPECGUI);
         if (pSpecUI && pSpecUI->IsVisible() && ipanel()->IsMouseInputEnabled(pSpecUI->GetVPanel()))
             SetMouseInputEnabled(true);
+
+        if (mom_mapfinished_movement_enable.GetBool())
+        {
+            g_pClientMode->GetViewportAnimationController()->StopAnimationSequence(this, "FadeOutMapFinishedPanel");
+            g_pClientMode->GetViewportAnimationController()->StartAnimationSequence("FadeOutMapFinishedPanel");
+        }
+        else
+        {
+            g_pClientMode->GetViewportAnimationController()->StopAnimationSequence(this, "FadeOutMapFinishedPanel");
+        }
```

Comment:
Both branches call stop animation for the `"FadeOutMapFinishedPanel"` animation, could be pulled out to above these two.